src/gm.datepickerMultiSelect.js

```diff
@@ -48,7 +48,7 @@ SOFTWARE.
 
 					/* Fires when date is selected or when month is changed. */
 					scope.$watch(function () {
-						return ctrl.activeDate.getTime();
+						return ctrl.activeDate ? ctrl.activeDate.getTime() : false;
```

Comment:
If you don't set a proper activeDate (like this.activeDate = new Date()) at
controller, it throws an exception, because it is null or undefined.

So I added this just to not raise an exception.

I think returning false doesn't create another $watch.

Let me know if you want a plunker of this or if you want to I revert this.
Em 13 de out de 2015 13:33, "spongessuck" <notifications@github.com>
escreveu:

> In src/gm.datepickerMultiSelect.js
> <https://github.com/spongessuck/gm.datepickerMultiSelect/pull/29#discussion_r41890905>
> :
>
> > @@ -48,7 +48,7 @@ SOFTWARE.
> >
> >  					/* Fires when date is selected or when month is changed. */
> >  					scope.$watch(function () {
> > -						return ctrl.activeDate.getTime();
> > +						return ctrl.activeDate ? ctrl.activeDate.getTime() : false;
>
> Why this change? You don't want to watch 'false,' and it will silently
> fail if ctrl.activeDate isn't defined.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/spongessuck/gm.datepickerMultiSelect/pull/29/files#r41890905>
> .
>